tests/serde-se.rs

```diff
@@ -75,7 +70,23 @@ enum ExternallyTagged {
     Empty {},
 }
 
+/// Having both `#[serde(flatten)]` and `'static` fields in one struct leads to
+/// incorrect code generation when deriving `Deserialize`.
```

Comment:
Can we have a consistent way we can mark code which depends on fixing external bugs, such that they can be easily searched?  Maybe `TODO(external)` or something.

Reply:
Yes, I forgot to mark those places with a TODO, will do!

I used an usual `TODO` and `NOTE` markers -- there are not many of them and the todo-tree VSCode extension shows them very well.